### DISCUSSION TITLE
Master Branch Still Active?

### DISCUSSION THREAD

**Discussion by slitchampion** [Serene-Arc/bulk-downloader-for-reddit#907] [General]
There seem to be several bug fixes and feature updates on the development branch, but the master branch doesn't seem to have been updated for about six months. Is it advised to users to switch to the dev branch, or is there a merge expected soon? Or maybe I'm missing something?

Super love the program and all of the features. Thanks for all that you all have done.

**Comment by Serene-Arc** [comment]
There is a merge expected soon. I'll do that today. I've been meaning to close more issues but it seems like it might be better to just pull the trigger now.

**Reply by slitchampion** [reply]
Awesome! Started having some rate limiting issues yesterday, so came looking for updates. Eager to see what's been done sine January.

Thanks again.

**Reply by twentyonerooms87** [reply]
Having the same issues with rate limiting which started yesterday as well.

**Reply by slitchampion** [reply]
Looks like the merge failed a couple of mandatory checks. Curious if any of the updates address some of the recent ratelimiting.

**Reply by Serene-Arc** [reply]
Not yet, I'm working on it.

**Reply by Fakeaccount12312** [reply]
Any update?

**Reply by Serene-Arc** [reply]
I've told Ali about the problem, now I'm waiting on him to update the repository secrets. I don't have the power to do that.

The development branch has updates that deal with the ratelimiting issue.

**Comment by SarahGillespie** [comment]
I got a "prawcore.exceptions.TooManyRequests: received 429 HTTP response" error today... sounds like the Reddit API changes hit this hard :( wishing you the best!

**Reply by Serene-Arc** [reply]
There are updates for that error on the development branch if you'd like to install that. Unfortunately they aren't on the master branch yet.
